src/app/backend.service.ts

```diff
@@ -47,7 +47,7 @@ export class BackendService extends BaseBackendService {
 
     // User has been identified, create a poem for them
     const endpoint = `${this.url}/api/create_poem/${userEmail}/${poemName}/${
-        poemText}/${generated}`;
+        poemText}/${generated ? 1 : 0}`;
```

Comment:
As a follow up to the other comment, this might look like:
`api/edit_poem/${userEmail}/${poemId? || ''}`

Reply:
Ok, I'll use that route for the edit endpoint.